### ISSUE TITLE
Request for Characters with "_m"

### ISSUE THREAD

**Issue by isjang08** [DeepMotionEditing/deep-motion-editing#139]
Hi, 
I'm a graduate student researching motion retargeting. I am excited about your work and paper. 
I ran your code and tested the network on new motion data with  Mixamo. 
I noticed characters with "_m" are not the same as them in Mixamo. 
Could you provide me these 4 characters for research purposes?

Thanks in advance.

**Comment by PeizhuoLi** [comment]
Thanks for your interest in our project. As we described in README and paper, we manually split three joints in `group A`. The "_m" suffix is used to denote the character that has been processed. You may use `datasets/split_joint.py` to get the character processed. For more details please check README.

**Comment by isjang08** [comment]
Thanks for the quick answer! It helps me for testing the network model.

I have one further question. 
Have you tested on validation data which is un-seen motion in the training process?
The pre-trained model and the model I trained seem not retargeting motion well on motion data not included in the training set.

**Comment by PeizhuoLi** [comment]
You are welcome : )

In our experiment we split the dataset into two parts (training and test) following [NKN](https://github.com/rubenvillegas/cvpr2018nkn). The model worked well on the test set.

If it's possible you can provide some example on the failure cases so I can understand better about the details.

**Comment by isjang08** [comment]
There was one mistake in me using the old npy file. It is working well now. 
Sorry for the confusion and thank you for your kind answer.
